README.md

```diff
@@ -99,20 +104,32 @@ Drop by our slack channel (#playback) on the [Video.js slack][slack-link].
 Get a copy of [videojs-http-streaming](#installation) and include it in your page along with video.js:
 
 ```html
-<video id=example-video width=600 height=300 class="video-js vjs-default-skin" controls>
+<video-js id=vid1 width=600 height=300 class="vjs-default-skin" controls>
```

Comment:
might be worth having a section why video-js, though, maybe just a FAQ about the late-init stuff in video.js instead?

Reply:
Yeah I'll add an explanation and also add `player.src()` as an alternate setup